src/component/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { getMissions } from '../redux/mission/mission';
+
+const Navbar = () => {
+  const { status } = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (status !== 'fetched') {
+      dispatch(getMissions());
+    }
+  }, []);
+
+  return (
+    <nav className="navBar">
+      <ul>
+        <li key={1}>
+          <NavLink to="/" className={(navData) => (navData.isActive ? 'active-link' : '')}>
+            Rocket
+          </NavLink>
+        </li>
+        <li key={2}>
+          <NavLink to="/Dragon" className={(navData) => (navData.isActive ? 'active-link' : '')}>
+            Dragon
+          </NavLink>
+        </li>
+        <li key={3}>
+          <NavLink to="/Mission" className={(navData) => (navData.isActive ? 'active-link' : '')}>
+            Mission
+          </NavLink>
+        </li>
+        <li key={4}>
+          <NavLink to="/Myprofile" className={(navData) => (navData.isActive ? 'active-link' : '')}>
+            My Profile
+          </NavLink>
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] I think you have misconfigured the styling, the `NavBar` is broken on the project I cloned, and the live version on `netlify`. Kindly make sure to add the proper styling and that the currently active page should be indicated with an underline.
![image](https://user-images.githubusercontent.com/33767951/147228753-bc7b5664-5b71-4091-aff4-c51daf6c0962.png)